src/Sylius/Behat/Context/Setup/OrderContext.php

```diff
@@ -119,9 +119,9 @@ public function __construct(
     }
 
     /**
-     * @Given the customer :customer placed an order :orderNumber
+     * @Given there is a customer :customer that placed an order :orderNumber
      */
-    public function theCustomerPlacedAnOrder(CustomerInterface $customer, $orderNumber)
+    public function aCustomerPlacedAnOrder(CustomerInterface $customer, $orderNumber)
```

Comment:
Should we write articles in camelCased method names? `customerPlacedOrder` instead of `aCustomerPlacedAnOrder` and `customerBoughtSingleProduct` instead of `theCustomerBoughtASingleProduct` looks cleaner to me. Typing `$this->custom...` will show me all actions that customer may perform, without thinking if I should look at `aCustom...` or `theCustom...`.

Moreover, we do not know in the method if it is a new customer or the one defined earlier, because it is the responsibility of parameters transformers to provide us customer instance in any case. We just get `customer`, nothing more or less.